frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -84,6 +86,55 @@ export default class Metadata extends React.Component<Props> {
         </Localized>;
     }
 
+    renderGroupComment(entity: Entity): React.Node {
+        if (!entity.group_comment) {
+            return null;
+        }
+
+        let comment = entity.group_comment;
+
+        return <Localized id='entitydetails-Metadata--group_comment' attrs={ { title: true } }>
+            <Property title='Group Comment' className='comment'>
+                <Linkify properties={ { target: '_blank', rel: 'noopener noreferrer' } }>
+                    { comment }
+                </Linkify>
+            </Property>
+        </Localized>;
+    }
+
+    state = { seeMore: false }
+
+    handleClick = () => {
+        this.setState({ seeMore: true });
+    };
```

Comment:
We should use state here. See more and the truncated resource comment should be reset every time you move to the next entity.

Reply:
I guess you meant "we should **not**"? State is actually fine, but it should be reset every time the entity changes. Also, since this state is specific to only the resource comment part, I think I would recommend to move that part into a new component. It can be a very simple component (maybe even using React Hooks? ).

Reply:
Yes, I mean we should NOT. And I'm wrong, thanks for the correction and sorry for the confusion, April!